tests/network/service_mesh/test_service_mesh.py

```diff
@@ -57,6 +61,24 @@ def test_authentication_policy_from_mesh(
             service_app_name=httpbin_service_service_mesh.app_name,
         )
 
+    @pytest.mark.ipv4
+    @pytest.mark.polarion("CNV-12181")
+    @pytest.mark.single_nic
+    @pytest.mark.dependency(
+        name="test_authentication_policy_from_mesh_post_migration",
+        depends=["test_authentication_policy_from_mesh"],
+    )
+    def test_authentication_policy_from_mesh_post_migration(
```

Comment:
It is not "post" migration.
Maybe "on", to reflect it survives migration in general (during + post).

Reply:
Ack. Previously we used the term over. Makes sense? e.g.:
test_authentication_policy_from_mesh_over_migration

Reply:
Sounds the same to me, go ahead.

Reply:
Ack.